journal/src/main/scala/com/github/j5ik2o/akka/persistence/dynamodb/journal/DynamoDBJournal.scala

```diff
@@ -341,7 +349,7 @@ class DynamoDBJournal(config: Config) extends AsyncWriteJournal with ActorLoggin
     implicit val ec: ExecutionContext = defaultExecutionContext
 
     val pid        = PersistenceId(persistenceId)
-    val context    = MetricsReporter.newContext(UUID.randomUUID(), pid)
+    val context    = Context.newContext(UUID.randomUUID(), pid)
     val newContext = metricsReporter.fold(context)(_.beforeJournalAsyncReplayMessages(context))
     val future = journalDao
```

Comment:
Change to def.

Reply:
fixed.
https://github.com/j5ik2o/akka-persistence-dynamodb/pull/943/commits/91e2814a4fcf841b337569bf886e735fd5f6d93e#diff-761788fe0403c57a933e6fcf54d2517b298b1a0afcc9e4ef3586ac5b03c3289eR355